clang/include/clang/AST/ASTTypeTraits.h

```diff
@@ -307,7 +307,7 @@ class DynTypedNode {
 
   /// For nodes which represent textual entities in the source code,
   /// return their SourceRange.  For all other nodes, return SourceRange().
-  SourceRange getSourceRange() const;
+  SourceRange getSourceRange(bool IncludeQualifier = false) const;
```

Comment:
I have a mild preference for this being `getSourceRangeIncludingQualifier` (that is, an overload instead of a default arg)?  Unless it results in an `if/else` tree elsewhere.